doc/source/serve/performance.md

```diff
@@ -20,20 +19,24 @@ We are continuously benchmarking Ray Serve. The metrics we care about are latenc
 - Ray Serve’s latency overhead is single digit milliseconds, around 1-2 milliseconds on average.
 - For throughput, Serve achieves about 3-4k queries per second on a single machine (8 cores) using 1 HTTP proxy actor and 8 replicas performing noop requests.
 - It is horizontally scalable so you can add more machines to increase the overall throughput. Ray Serve is built on top of Ray,
-  so its scalability is bounded by Ray’s scalability. Please check out Ray’s [scalability envelope](https://github.com/ray-project/ray/blob/master/release/benchmarks/README.md)
+  so its scalability is bounded by Ray’s scalability. Please see Ray’s [scalability envelope](https://github.com/ray-project/ray/blob/master/release/benchmarks/README.md)
   to learn more about the maximum number of nodes and other limitations.
 
+We run long-running benchmarks nightly. You can check out [our workloads in the Ray repository](https://github.com/ray-project/ray/tree/master/release/serve_tests/workloads).
+
+You can see our nightly benchmark runs [here](https://buildkite.com/ray-project/release-tests-branch/builds?branch=master). They're labeled as "Nightly tests." For example, [here's the nightly benchmark run from July 22nd, 2022](https://buildkite.com/ray-project/release-tests-branch/builds/806). You can see the results by clicking "Serve tests" and choosing a workload.
+
 You can check out our [microbenchmark instructions](https://github.com/ray-project/ray/blob/master/python/ray/serve/benchmarks/README.md)
 to benchmark on your hardware.
 
 ## Debugging performance issues
 
 The performance issue you're most likely to encounter is high latency and/or low throughput for requests.
 
-If you have set up [monitoring](serve-monitoring) with Ray and Ray Serve, you will likely observe the following:
+Once you set up [monitoring](serve-monitoring) with Ray and Ray Serve, you can likely observe the following:
 
 - `serve_num_router_requests` is constant while your load increases
-- `serve_deployment_queuing_latency_ms` is spiking up as queries queue up in the background
+- `serve_deployment_processing_latency_ms` is spiking up as queries queue up in the background
```

Comment:
#15065 removed `serve_deployment_queuing_latency_ms`, and the closest metric I found was `serve_deployment_processing_latency_ms`. @simon-mo @edoakes Does this replacement make sense?

Reply:
Yes. Make sense. Same metrics basically.

Reply:
Yes. I think the metrics is basically queuing + processing.